st3/sublime_lib/settings_dict.py

```diff
@@ -181,10 +182,7 @@ def onchange():
 
         key = str(uuid4())
         self.settings.add_on_change(key, onchange)
-        return key
-
-    def unsubscribe(self, key):
-        self.settings.clear_on_change(key)
+        return lambda: self.settings.clear_on_change(key)
```

Comment:
I usually prefer `functools.partial` over lambdas because they are more robust, though in this particular case there shouldn't be any problems.